utilities/ride-model-util/src/main/java/com/adobe/ride/utilities/model/ModelObject.java

```diff
@@ -740,9 +740,8 @@ private JSONObject buildObjectNode(String pathToParent, JSONObject obj) {
           Object genValue = null;
           try {
             if (currentkey.equals("otherArticles")) {
-              System.out.println("Debug");
+              genValue = generateNodeValue(null, currentkey, propertyDef);
```

Comment:
Hello, and thank you for taking a look at this!  However, the conditional here is only related to the debug statement.  The generateNodeValue call needs to exist alone in the try statement.  The conditional and system out needs to be removed, but value generation statement needs to stay.

Thank you!

Reply:
Hi @tedcasey just did it!